ui/src/adapters/api/credentials.ts

```diff
@@ -149,7 +149,7 @@ export async function linkUpdate({
   };
 }): Promise<APIResponse<string>> {
   try {
-    const response = await axios<{ message: string }>({
+    const response = await axios({
```

Comment:
Wasn't this fixed in the other PR? why is the fix also here?

Reply:
This one comes from develop not revoke credentials

Reply:
Of course, my question is: why do u fix this twice?

Reply:
To ensure that I don't forget to import the parser when having a conflict at the merge